### PR TITLE
Don't install static archives

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks Uwe! 🙏

FWIW this looks good. Just want to clear out the CI issues

**Comment by xhochy** [comment]
Problem still persists: UnicodeEncodeError: 'utf-8' codec can't encode character '\udce4' in position 5: surrogates not allowed

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by xhochy** [comment]
Depends on https://github.com/conda-forge/ucx-split-feedstock/pull/154

**Comment by jakirkham** [comment]
Thanks Uwe! 🙏

Have merged that PR. Could you please fix the conflicts here?

File: conda-forge.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 azure:
   store_build_artifacts: true
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```
Comment (jakirkham) [inline conda-forge.yml:5]: Let's stick to native builds. Cross-compilation doesn't work here

File: conda-forge.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 conda_build:
   pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-provider:
-  linux_aarch64: default
-  linux_ppc64le: default
```
Comment (jakirkham) [inline conda-forge.yml:11]: Was this meant to be dropped?

**Review by jakirkham — COMMENTED** [review]
Thanks Uwe! 🙏

Made a few comments below

**Comment by xhochy** [comment]
Reverted to native compilation :( I was annoyed by Travis but seems like we need to stick with it then.

**Comment by jakirkham** [comment]
Yeah sorry. Had tried this before too. Unfortunately it appears to be a little complicated to switch 😞

Let's stick with the current changes here and revisit cross-compilation separately 👍

**Comment by jakirkham** [comment]
Restarting Travis CI build that was stuck in queue. The automerge bot may complain, but we can ignore it

**Comment by jakirkham** [comment]
Thanks Uwe! 🙏

**Pull request merged**